app/models/reminder.js

```diff
@@ -0,0 +1,9 @@
+import DS from 'ember-data'
+
+export default DS.Model.extend({
+  title: DS.attr('string'),
+  date: DS.attr('date'),
+  notes: DS.attr('string'),
+  pinned:
```

Comment:
This can be on one line. This won't keep your PR from being merged in but keep an eye on formatting. Also the pinned property should have a default value of false - lets fix that for the next PR.